docker/llm/finetune/xpu/docker/qlora_finetune_xpu.py

```diff
@@ -0,0 +1,58 @@
+import torch
```

Comment:
Is it possible to use the original example. 

I am afraid this one will be easily forgot when we do future update.

Reply:
done